scene/2d/tile_map_layer.cpp

```diff
@@ -29,7 +29,8 @@
 /**************************************************************************/
 
 #include "tile_map_layer.h"
-
+#include "editor/editor_node.h"
```

Comment:
Scene code should not use editor code, you're not even using this anywhere I can see?